testsuite/tests/parsing/prefix_op.ml

```diff
@@ -0,0 +1,36 @@
+(* TEST_BELOW
+(* Blank lines added here to preserve locations. *)
+
+
+
+
+*)
```

Comment:
This is a new test, so you don't need those blank lines -- I guess you copied it from somewhere else.

Reply:
Fixed! Indeed, I didn't know how to write a test and so took an other one as a base. Looks much better with `-dno-locations -stop-after parsing`!